VidCoder/Resources/PickerRes.resx

```diff
@@ -250,4 +250,7 @@
     <value>Forced only</value>
     <comment>Label for checkbox in Picker window to choose only Forced subtitles of the chosen subtitle track.</comment>
   </data>
+  <data name="OverrideNameFormatLabel" xml:space="preserve">
+    <value>Available arguments: {source} {title} {range} {preset} {date} {time} {quality} {parent} {titleduration}</value>

```

Comment:
I think this value should be Available arguments: {0} and have the {source}, etc hard-coded in the code, as that part should not get translated.

Reply:
Thanks for feedback. Hmm. It depends. I know the {placeholders} are part of code and you could possibly change/extend it "one day". On the other hand this solution gives you possibility to explain some particular {placeholder}, e.g. {range} and {parent} are not obvious from my point of view.

Reply:
I'd rather link to the webpage that explains all of them if I want to explain them. The purpose of this string is simply to list all of the available ones tokens.